integr8ly/integreatly-operator
# INTLY-3932: Move manifests files to the integreatly operator repo

## What
The following changes moves the manifest files from this [repository](https://github.com/integr8ly/manifests) to the Integreatly operator repo. All of the manifest files are now located in the `manifests/` folder with only the latest version of each CSV included.

A new Makefile called `manifests.mk` has been created within the `manifests/` folder. This has all of the targets required to push all of or push individual manifest files to a specified app registry. These targets are imported to the main `Makefile` which is located in the root folder of this repository.

A new target has also been added to the main `Makefile` to push the integreatly operator and all of the product CSVs. This can be ran by using `make push/all REPO=<repo>`

## Verification
1. Ensure that all of the manifests contains the latest CSV version from this [repository](https://github.com/integr8ly/manifests)
2. Run `make push/all REPO=<repo>`
3. Verify that all product CSVs has been pushed to the specified app registry
4. Verify that the integreatly operator CSV has been pushed to the specified app registry

**NOTE**: This should not affect the e2e tests
